assets/components/src/tabbed-navigation/index.js

```diff
@@ -7,15 +7,10 @@
  */
```

Comment:
On Mobile (iPhone XS) tabbed navigation with many tabs gets tight:

<img width="1595" alt="Screen Shot 2019-12-11 at 11 11 48 PM" src="https://user-images.githubusercontent.com/1477002/70682103-bb705100-1c6b-11ea-822e-52c73b459166.png">

Reply:
I've changed the behaviour of this on small screens in 1ace43e